parse.go

```diff
@@ -100,29 +100,32 @@ func getRawPayloadFromX509(cert *x509.Certificate) ([]byte, error) {
 			return ext.Value, nil
 		}
 	}
+
 	return nil, trace.NotFound(
 		"certificate does not contain extension with license payload")
 }
 
 // parseCertificatePEM parses the concatenated certificate/private key in PEM format
 // and returns certificate and private key in decoded DER ASN.1 structure
-func parseCertificatePEM(certPEM string) ([]byte, []byte, error) {
-	var certificateBytes, privateBytes []byte
+func parseCertificatePEM(certPEM string) ([]byte, []byte, []byte, error) {
```

Comment:
```suggestion
func parseCertificatePEM(certPEM string) (certPEM []byte, keyPEM []byte, anonKeyPEM []byte, error) {
```

Let's name the return values since they're all the same type. It will be easier to tell which is which.

Reply:
Done